Documentation/workflow/DevelopmentTips.md

```diff
@@ -373,3 +373,69 @@ steps rely on having the Android NDK installed.
         (gdb) set sysroot /tmp/gdb-symbols
         (gdb) set solib-search-path /tmp/gdb-symbols
         (gdb) target remote :50999
+
+# .NET 6 Tips
+
+## Finding Mono runtime packs
+
+`main` builds of the Mono runtime packs are on the following NuGet
+feed, such as this `nuget.config`:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
+Internal MS users can view packages on Azure DevOps here:
+
+* https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet6
+
+To find the commit of a given package version, locate the `.nuspec`
+file inside the package, and look for:
+
+```xml
+<repository type="git" url="https://github.com/dotnet/runtime" commit="7bd472498e690e9421df86d5a9d728faa939742c" />
+```
+
+## Testing different Mono runtime pack versions
+
+One common scenario that comes up -- how does one test a specific
+dotnet/runtime build along with a .NET 6 Android application?
+
+The simplest way would be to copy individual files on top of the NuGet
```

Comment:
I'm not sure I'd call this "simple", as there are *lots* of files in there, most of which is inscrutable.  My `$HOME/.nuget/packages/microsoft.netcore.app.runtime.mono.android-arm` contains 7 directories; which one should I update?  How do I figure that out?  (Answer: by reading an existing build log, and updating whichever version was implicated.  Requires reading build logs!)

`$HOME/.nuget/packages/microsoft.netcore.app.runtime.mono.android-arm/6.0.0-preview.7.21352.16` contains 230 files as well.  How is anyone supposed to know which files to update?

This isn't "simple".  This is the "advanced" "I think i know what I'm doing, maybe" scenario.  It just *looks* simple.